code/__DEFINES/xeno.dm

```diff
@@ -653,3 +657,6 @@
 #define NOT_WEEDABLE 0
 #define SEMI_WEEDABLE 1
 #define FULLY_WEEDABLE 2
+
+// Royal
+#define ROYAL_SCREECH_BUFFED  	1
```

Comment:
probably would be wise to give this a different name since it might get confused with ROYAL_SCREECH_BUFF
may as well move it up next to those royal_screech defines so they're all together
```suggestion
#define ROYAL_SCREECH_ENHANCED  	1
```

Reply:
note this suggestion won't replace all instances of the former name